src/logic/logic.shaii.ts

```diff
@@ -458,6 +458,58 @@ export function spongify(sentence: string, capsOdd: boolean = true): string {
   return newSentence;
 }
 
+/**
+ * 
+ * @param sentence the sentence to brainfuck 
+ * @author Qexat // based (copied tbh lol) on HelgeFox's work <https://github.com/helgeh/brainfuck-text>
+ */
+ export function textToBrainfuck(sentence: string): string {
+	function closest(num: number, arr: number[]) {
+	  var arr2 = arr.map((n) => Math.abs(num - n));
+	  var min = Math.min.apply(null, arr2);

```

Comment:
please replace all the `var`s with `const` / `let`

Reply:
already fixed by the commit dc45f1c?